src/react/src/components/Contribute/ProductionLocationInfo.jsx

```diff
@@ -0,0 +1,527 @@
+                        }
+                        error={nameTouched && !validate(inputName)}
+                    />
+                </div>
+                <div className={classes.inputSectionWrapStyles}>
+                    <Typography component="h2" className={classes.titleStyles}>
+                        Address
+                    </Typography>
+                    <Typography
+                        component="h4"
+                        className={classes.subTitleStyles}
+                    >
+                        Enter the address of the production location. We will
+                        use this to plot the location on a map.
+                    </Typography>
+                    <TextField
+                        id="address"
+                        className={classes.textInputStyles}
+                        value={inputAddress}
+                        onChange={handleAddressChange}
+                        placeholder="Enter the full address"
+                        variant="outlined"
+                        aria-label="Enter the address"
+                        InputProps={{
+                            classes: {
+                                input: `${classes.searchInputStyles}
+                                ${
+                                    addressTouched &&
+                                    !validate(inputAddress) &&
+                                    classes.errorStyle
+                                }`,
+                                notchedOutline: classes.notchedOutlineStyles,
+                            },
+                            inputProps: {
+                                type: 'text',
+                            },
+                        }}
+                        helperText={
+                            addressTouched &&
+                            !validate(inputAddress) && <InputHelperText />
+                        }
+                        error={addressTouched && !validate(inputAddress)}
+                    />
+                </div>
+                <div className={classes.inputSectionWrapStyles}>
+                    <Typography component="h2" className={classes.titleStyles}>
+                        Country
+                    </Typography>
+                    <Typography
+                        component="h4"
+                        className={classes.subTitleStyles}
+                    >
+                        Enter the country where the production site is located.
+                    </Typography>
+                    <StyledSelect
+                        id="countries"
+                        name="Country"
+                        aria-label="Country"
+                        label={null}
+                        options={countriesOptions || []}
+                        value={inputCountry}
+                        onChange={handleCountryChange}
+                        className={`basic-multi-select notranslate ${classes.selectStyles}`}
```

Comment:
className already exists in the StyledSelect.

Reply:
deleted